docs/docs/development/local-toolchain/setup/docker/cli.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Dev Container CLI
+sidebar_label: Dev Container CLI
+---
+
+### Installing Development Tools
+
```

Comment:
Would this also work with Podman? I know a number of people prefer to use it due to not needing admin privileges or smth.

Reply:
So I never really used Podman before, but I was trying around a bit to find a way of using it for local building. I had success by using the Dockerfile using the steps listed below. By now I haven't tried using `devcontainer-cli` but I am looking into it.

1. `podman machine init <machine-name>`: Initialize a new virtual machine called `<machine-name>`
2. `podman machine start <machine-name>`: Start the virtual machine
3. `podman build -t <container-name> -f Dockerfile </path/to/.devcontainer>`: Build a new container called `<container_name>`
4. Start the container and mount the volumes like it is done in the Docker approach:
```sh
podman run -it \
  --workdir /workspaces/zmk \
  -v /path/to/zmk:/workspaces/zmk \
  -v /path/to/zmk-config:/workspaces/zmk-config \
  -v /path/to/zmk-modules:/workspaces/zmk-modules \
  -p 3000:3000 \
  <container-name> /bin/bash
```

From here on I was able to run all the commands needed to build my keyboard:
```sh
west init -l /app
west update
west build -p -b nice_nano_v2 -d build/left -- -DSHIELD=corne_left -DZMK_CONFIG=/workspaces/zmk-config -DZMK_EXTRA_MODULES=/workspaces/zmk-modules/zmk-auto-layer
```

Reply:
The Podman docmentation is now added as well.